src/index.html

```diff
@@ -5,8 +5,398 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        class="logo_image">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__item"  data-qa="nav-hover">
+            <a class="nav__link" href="#">Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+    <main class="main">
+      <div class="catalog">
+
+        <div class="card" data-qa="card">
+          <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+          >
+
+          <span class="card__name">
```

Comment:
```suggestion
          <h2 class="card__name">
```
Consider leveraging semantic tags